script.js

```diff
@@ -0,0 +1,230 @@
+  },
+  modal3: {
+    title: 'Tonic',
+    subt: ['CANOPY', 'Back End Dev', '2015'],
+    desc: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+    img: 'img/SnapshootPortfolio(3).png',
+    tech: ['html', 'css', 'javaScript'],
+    live: 'https://mutinhiri.github.io/My-portfolio/',
+    source: 'https://github.com/mutinhiri/portfolio-setup',
+  },
+  modal4: {
+    title: 'Multi-Post Stories',
+    subt: ['CANOPY', 'Back End Dev', '2015'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    img: 'img/SnapshootPortfolio(4).png',
+    tech: ['html', 'css', 'javaScript'],
+    live: 'https://mutinhiri.github.io/My-portfolio/',
+    source: 'https://github.com/mutinhiri/portfolio-setup',
+  },
+};
+
+// const modalSubt = document.getElementById('modalSubt');
+// const modalTech = document.getElementById('modalTech');
+
+// // // function addCounter() {
+// //   const counter = document.createElement('img');
+// //   counter.src = 'img/Counter.svg';
+// //   counter.alt = 'counter';
+// //   modalSubt.appendChild(counter);
+// // }
+
+// function addSubt(modal) {
+//   const firstDiv = document.createElement('div');
+//   firstDiv.className = 'font500 black font18-24';
+//   modalSubt.appendChild(firstDiv);
+//   [firstDiv.innerHTML] = modalObj[modal].subt;
+//   modalObj[modal].subt.slice(1).forEach((i) => {
+//     addCounter();
+//     const otherDiv = document.createElement('div');
+//     otherDiv.className = 'gray font18-24';
+//     modalSubt.appendChild(otherDiv);
+//     otherDiv.innerHTML = i;
+//   });
+// }
+
+// function addTechs(modal) {
+//   modalObj[modal].tech.forEach((i) => {
+//     const li = document.createElement('li');
+//     li.appendChild(document.createTextNode(i));
+//     modalTech.appendChild(li);
+//   });
+// }
+
+// function clearModal() {
+//   modalTech.innerHTML = '';
+//   modalSubt.innerHTML = '';
+// }
+
+function popModal(modal) {
```

Comment:
Please use the background color from the design page. Moreover, It does not show project buttons and technologies on all screens, add a cross icon so that the user is able to close the pop-up, and enable vertical scroll. Kindly see the screenshot below and fix the design issues.